site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -0,0 +1,24 @@
+{% if peer %}
```

Comment:
Instead of having a server-side check for peer grading what you can do is always load this within a div and then hide it by default (display:none) and then perfom a javascript check to see if there are any peer components and if there are you can unhide.